### DISCUSSION TITLE
How do you access runtime.js from App.svelte?

### DISCUSSION THREAD

**Discussion by MerreM** [wailsapp/wails#4200] [Q&A]
### Have you read the Documentation Contribution Guidelines?

- [X] I have read the [Documentation Contribution Guidelines](https://wails.io/community-guide#ways-of-contributing).

### Description

I'm trying `import EventsOn from "../wailsjs/runtime/runtime.js";`

Which results in the below error;

`RollupError: "default" is not exported by "wailsjs/runtime/runtime.js", imported by "src/App.svelte".`

I can access the methods via window.runtime - but there must be a way to import them, surely?

### Self-service

- [x] I'd be willing to address this documentation request myself.

**Comment by samfweb** [comment]
Try without the .js: 
`import WailsRuntime from "../wailsjs/runtime/runtime";`

Alternatively:
`import {EventsOn, EventsOff } from "../wailsjs/runtime/runtime";`
